docker/test/stress/run.sh

```diff
@@ -284,6 +284,11 @@ then
 
     rm -rf /var/lib/clickhouse/*
 
+    # Make BC check more funny by forcing Ordinary engine for system database
```

Comment:
![image](https://user-images.githubusercontent.com/3684697/175816308-43294c19-0ad5-4232-9d0d-2a8c710767d1.png)

Reply:
@alesapin Is that you?